datumaro/plugins/sampler/relevancy_sampler.py

```diff
@@ -165,7 +165,7 @@ def _load_inference_from_subset(extractor, subset_name):
             width, height = item.media.size
             data_df["Width"].append(width)
             data_df["Height"].append(height)
-            data_df["ImagePath"].append(item.media.path)
+            data_df["ImagePath"].append(getattr(item.media, "path", None))
```

Comment:
I guess we would rather add `path` property which returns `None` to the base media class.

Reply:
Yes, indeed. `path` is way more used than I thought across Datumaro. Lets make it in another PR.